Editor/ArcadiaProjectInitialization.cs

```diff
@@ -0,0 +1,33 @@
+﻿using UnityEngine;
+using UnityEditor;
+
+namespace Arcadia
+{
+    [InitializeOnLoad]
+    public class ArcadiaProjectInitialization
+        // Starts with an "A" because Editor scripts are initialized alphabetically 
```

Comment:
That gets serialized into the `ProjectSettings` folder. I'd rather not depend on that. If it sorts alphabetical by default, leaning on that makes sense. It's just such a funny architectural decision.